packages/snaps-execution-environments/src/common/endowments/endowments.test.browser.ts

```diff
@@ -154,8 +157,6 @@ describe('endowments', () => {
       },
       AbortSignal: {
         endowments: { AbortSignal },
-        // @ts-expect-error - `abort()` method exists in browser, but not in
```

Comment:
Huh, wondering why this was an issue in the first place? Doesn't Node.js have this method?

Reply:
Likely a transitive dependency bringing in declarations for this function, which is now removed or bumped.